src/go/web/workflow.go

```diff
@@ -32,8 +32,17 @@ func ApplyWorkflow(w http.ResponseWriter, r *http.Request) error {
 		role  = ctx.Value("role").(rbac.Role)
 		vars  = mux.Vars(r)
 		scope = vars["branch"]
+		q     = r.URL.Query()
+		tags  = ""
 	)
 
+	// currently querries only are used to pass tags. However, this
+	// is extensible for future fields as well.
+	for _, s := range q["tag"] {
+		tags += s + ","
+	}
+	tags = strings.TrimSuffix(tags, ",")
+
```

Comment:
I would suggest moving this down below the `if !role.Allowed` check just to avoid unnecessary work if the user can't create a workflow anyway.

Also, you can condense this down to a single line and get the same result.
```
tags := strings.Join(q["tag"], ",")
```
If `q` does not have a `tag` key, an empty slice will be returned. Doing a `strings.Join` on an empty slice simply results in an empty string.